src/components/DpContextualHelp/DpContextualHelp.vue

```diff
@@ -24,6 +25,12 @@ export default {
 
   directives: {
     tooltip: Tooltip
+  },
+
+  computed: {
+    ariaLabel () {
+      return de.contextualHelp
+    }
```

Comment:
I think a `data` is sufficent enough here. That avoids unnecessary eventlistener as far as I understand vue.
(would affect several places)

Reply:
Thanks, changed in b2fd42ac0892bcb2d81712bb0f1a578e2ca46ae3 (I simply did it for all demosplan-ui)